File: demos/KendoCRUDService/KendoCRUDService/Controllers/EventsController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using KendoCRUDService.Models;
using KendoCRUDService.Common;
using System.Web.Script.Serialization;

namespace KendoCRUDService.Controllers
{
    public class EventsController : Controller
    {
        public ActionResult Index()
        {
            return this.Jsonp(EventsRepository.All());
        }                       
        
        public JsonResult Update()
        {
            var models = this.DeserializeObject<IEnumerable<EventModel>>("models");
            if (models != null)
            {
                EventsRepository.Update(models.First());
            }
            return this.Jsonp(models);
        }
        
        public ActionResult Destroy()
        {
            var events = this.DeserializeObject<IEnumerable<EventModel>>("models");

            if (events != null)
            {
                EventsRepository.Delete(events.First());
            }
            return this.Jsonp(events);
        }
        
        public ActionResult Create()
        {
            var events = this.DeserializeObject<IEnumerable<EventModel>>("models");
            if (events != null)
            {
                EventsRepository.Insert(events.First());
            }
            return this.Jsonp(events);
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/map/EventsController.java
package com.kendoui.spring.controllers.map;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

@Controller("map-events-controller")
@RequestMapping(value="/map/")
public class EventsController {
    @RequestMapping(value = {"/events"}, method = RequestMethod.GET)
    public String index() {
        return "map/events";
    }
}

